### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Daniil-A4**

### CODE REVIEW

File: src/formatDate.js [hunk 1]
```diff
@@ -50,7 +50,66 @@
...
+      yearIndex = i;
+      break;
+    } else if (fromFormat[i].includes('YYYY')) {
+      yearIndex = i;
+      break;
+    }
+  }
+
+  const year = dateParts[yearIndex];
+  let year2 = '';
```
Comment (maxim2310) [inline src/formatDate.js:68]: use a meaningful name for the variable

File: src/formatDate.js [hunk 2]
```diff
@@ -50,7 +50,66 @@
...
+
+  if (year.length === 4) {
+    year2 = year.slice(2);
+  } else {
+    year2 = year;
+  }
+
+  const year4 = year2 < 30 ? `20${year2}` : `19${year2}`;
```
Comment (maxim2310) [inline src/formatDate.js:76]: use a meaningful name for the variable

File: src/formatDate.js [hunk 3]
```diff
@@ -50,7 +50,66 @@
...
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    if (fromFormat[i].includes('YY')) {
+      yearIndex = i;
+      break;
+    } else if (fromFormat[i].includes('YYYY')) {
+      yearIndex = i;
+      break;
+    }
+  }
```
Comment (maxim2310) [inline src/formatDate.js:65]: better use indexOf without loop for

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just couple tiny things needs to be fixed before approval)

**Review by anastasiiavorobiova — APPROVED** [review]
Great work!

**Pull request closed without merging**